src/azul/__init__.py

```diff
@@ -57,15 +57,19 @@ def es_endpoint(self) -> Optional[Netloc]:
             host, _, port = es_endpoint.partition(':')
             return host, int(port)
 
-    def es_endpoint_env(self, es_endpoint: Optional[Netloc], es_instance_count: Optional[int]) -> Mapping[str, str]:
+    def es_endpoint_env(self,
+                        es_endpoint: Optional[Netloc] = None,
+                        es_instance_count: Optional[int] = None
+                        ) -> Mapping[str, str]:
         if es_endpoint is None:
-            return {}
+            host, port = "${var.azul_es_domain.endpoint}", 443
+            es_instance_count = "${var.azul_es_domain.cluster_config[0].instance_count}"
```

Comment:
Brittle and hacky. An acceptable solution to this ticket should not embed TF syntax in `azul.config`, especially not TF syntax that hard-codes TF variable names. TF config can depend on `azul.config` but not the other way around.